frame/bounties/README.md

```diff
@@ -0,0 +1,118 @@
+# Treasury Module
+
+The Treasury module provides a "pot" of funds that can be managed by stakeholders in the
+system and a structure for making spending proposals from this pot.
+
+- [`treasury::Trait`](https://docs.rs/pallet-treasury/latest/pallet_treasury/trait.Trait.html)
+- [`Call`](https://docs.rs/pallet-treasury/latest/pallet_treasury/enum.Call.html)
+
+## Overview
+
+The Treasury Module itself provides the pot to store funds, and a means for stakeholders to
+propose, approve, and deny expenditures. The chain will need to provide a method (e.g.
+inflation, fees) for collecting funds.
+
+By way of example, the Council could vote to fund the Treasury with a portion of the block
+reward and use the funds to pay developers.
+
+### Tipping
```

Comment:
Need to clean up docs

Reply:
Thanks for the comment, its my bad, overlooked readme.md